hack/test-cmd.sh

```diff
@@ -568,6 +568,13 @@ echo "scale: ok"
 
 oc process -f examples/sample-app/application-template-dockerbuild.json -l app=dockerbuild | oc create -f -
 wait_for_command 'oc get rc/database-1' "${TIME_MIN}"
+oc rollback database --to-version=1 -o=yaml
```

Comment:
Can you make a separate stanza for `rollback` and add the `rollback: ok` message in the end ?